tstest/integration/integration_test.go

```diff
@@ -47,8 +46,6 @@ import (
 	"tailscale.com/version"
 )
 
-var verbose = flag.Bool("verbose", false, "verbose debug logs")
```

Comment:
How about we just rename this flag to something else, like "debug"? I don't want the normal "go test -v" level verbosity to include this much spam.

Reply:
@simenghe and I landed here because he couldn't find the client logs anywhere. It took me a few minutes to figure out why not, particularly since all the other logs are present. I'd rather accept the `-v` spam in exchange for discoverability.